SimplenoteUITests/NoteList.swift

```diff
@@ -188,6 +202,11 @@ class NoteListAssert {
         XCTAssertEqual(actualNotesNumber, expectedNotesNumber, numberOfNotesInAllNotesNotExpected)
     }
 
+    class func noteHeight(_ note: NoteData, _ noteHeight: CGFloat) {
```

Comment:
Nitpick / Readability idea.

```suggestion
    class func note(_ note: NoteData, hasHeight height: CGFloat) {
```

This would be used as `NoteListAssert.note(godzilla, hasHeight: 123.0)`, which I think reads nicely. But... it may just be my personal preference, so no big deal. Just tossing it there to play around with the language.

Reply:
I'm for that, because I wanted to build other methods using `has...` (hasPin, hasContent...) Thank you!